api/src/main/scala/com/avast/clients/rabbitmq/api/RabbitMQManualConsumer.scala

```diff
@@ -0,0 +1,20 @@
+package com.avast.clients.rabbitmq.api
+
+import scala.language.higherKinds
+
+trait RabbitMQManualConsumer[F[_], A] {
```

Comment:
Usually these are called `push` and `pull` models in streaming so I would vote for `RabbitMQPullConsumer`.

Reply:
Agreed, the underlying RMQ client Java API docs also refer to retrieving individual messages as the Pull API.

Reply:
Done